prow/github/hmac.go

```diff
@@ -46,3 +79,40 @@ func PayloadSignature(payload []byte, key []byte) string {
 	sum := mac.Sum(nil)
 	return "sha1=" + hex.EncodeToString(sum)
 }
+
+// extractHmacs returns all *valid* HMAC tokens for given repository/organization. It iterates over full tree as necessary (repo->org->global)
```

Comment:
The second part of this comment implies that it will match at any level, which isn't true. Also, considered hard wrapping to maintain reasonable line lengths.

```suggestion
// extractHmacs returns all *valid* HMAC tokens for given repository/organization.
// It considers only the tokens at the most specific level configured for the given repo.
```

Reply:
It says "as necessary" .. I know its not super clear. If you have a verbiage in mind do share.

Reply:
I suggested verbiage above.